models/sales/hubspot__deals.sql

```diff
@@ -36,6 +48,11 @@ with deals as (
     left join owners
         using (owner_id)
 
+    {% if var('hubspot_company_enabled', True) %}
+    left join companies 
+        using (company_id)
```

Comment:
It doesn't seem like `deals` contains a company_id. It may be lost within the `properties_` fields but I could not tell from our dataset. It also seems Circle failed on this step.

Reply:
ah my b -- this should join on deal_id